examples/mutators/apply-setters/README.md

```diff
@@ -0,0 +1,22 @@
+# Apply Setters
+
+The `apply-setters` KRM config function applies setter values to resource fields
```

Comment:
@mengqiy Follow up on other threads. Please follow up with an ideal example of a README in an example package. Other functions should follow that eample